pkg/devfile/adapters/common/utils.go

```diff
@@ -61,6 +61,9 @@ const (
 	// BinBash The path to sh executable
 	BinBash = "/bin/sh"
 
+	// FwdSlash is the path separator in linux containers
+	FwdSlash = "/"
```

Comment:
I don't think we need to store this in a variable as it is used in only one place and is only one character

Reply:
I did it because if you search for `"/"` in the odo repo, its all over the place. Some place has var defined, some are using direct strings. So I thought if I did it this way, it'd be at least consistent going forward. But I can remove it, if you think otherwise.

Reply:
I found only a few places where a variable has been created mostly in the URL package for URL paths. I would suggest using it directly as it is a single character string and probably won't change in the future or moving this variable to a general util package like /pkg/util/util.go for use across odo.

Reply:
updated. :)